src/Nox.Generator/Domain/EntityMetaGenerator.cs

```diff
@@ -21,63 +25,74 @@ public void Generate(SourceProductionContext context, NoxSolutionCodeGeneratorSt
         foreach (var entity in codeGeneratorState.Solution.Domain.Entities)
         {
             context.CancellationToken.ThrowIfCancellationRequested();
-
-            var propertiesWithSource = entity.GetAllMembers()
-                .Select( t => new { TypeDef = t.Value, Source = GenerateStaticTypeOptions(t.Value, codeGeneratorState.Solution) })
-                .Where( t => t.Source != null)
+            
+            var typeDef = entity.GetAllMembers()
+                .Select( t =>  GenerateStaticTypeOptions(t.Value, codeGeneratorState.Solution) )
                 .ToList();
-
+            
             new TemplateCodeBuilder(context, codeGeneratorState)
                 .WithClassName(entity.Name)
                 .WithFileNamePrefix($"Domain.Meta")
                 .WithObject("entity", entity)
-                .WithObject("propertiesWithSource", propertiesWithSource)
+                .WithObject("typeDef", typeDef)
```

Comment:
we can improve readability, changing the name for example
```suggestion
                .WithObject("entitiesMetadata", entitiesMetadata)
```
in the template will be also clear what we are iterating

Reply:
Desired changes were applied